docs/_articles/en/vscode-desktop/java-setup.md

```diff
@@ -12,31 +12,29 @@ You can download the JDK from one of these locations.
 
 [Adoptium](https://adoptium.net) provides prebuilt OpenJDK binaries for free without authentication or complications.
 
-1. Navigate to [Adoptium](https://adoptium.net/?variant=openjdk17).
-2. Select **Temurin 17 (LTS)**.
-3. Click the **Latest Release** button to download.
-4. After the file has downloaded, open it and complete the installation steps.
+1. Navigate to [Adoptium](https://adoptium.net/?variant=openjdk21).
```

Comment:
The link directly points to jdk 21, can the step be updated to say 
Navigate to Adoptium jdk21 download page?